package.json

```diff
@@ -249,7 +247,7 @@
     "clean": "npm run -s clean:build && npm run -s clean:devdocs && npm run -s clean:public && npm run -s clean:packages",
     "clean:build": "npx rimraf build server/bundler/*.json .babel-cache",
     "clean:devdocs": "npx rimraf server/devdocs/search-index.js server/devdocs/proptypes-index.json server/devdocs/components-usage-stats.json",
-    "clean:packages": "npx rimraf packages/*/dist",
+    "clean:packages": "npx lerna run clean",
```

Comment:
Should we `--stream` this like build?

Reply:
sure. it just changes the output format. i don't have a strong preference between standard and stream.